doc/legacy/packaging-0.1.0.md

```diff
@@ -0,0 +1,29 @@
+# Yeast packaging for sale
+
+version = "0.1.0"
+
+## Overview
+
+This protocol outlines packaging of small yeast containers for sale
+
+## Prerequisites
+
+- Propagated yeast mass in a vessel
+- sterile containers
+- Sterile transfer kit
+- Sterile water, approximately 1/5 of yeast mass volume
+- Filling funnel, sterilized
+- Labels, bags, and other packaging components
+
+## Procedure
+
+1. Let yeast settle; if this is unreasonably slow, facilitate it by rapid cooling.
+2. Pour out and discard as much liquid phase as possible
+3. Add sterile water to vessel in small portions and re-suspend caked yeast into paste by gentle motion
+4. Pour yeast suspension into sterile containers through the funnel to avoid splashing the orifice or outer surfaces; close containers
```

Comment:
```suggestion
4. Pour yeast suspension into sterile containers through the funnel to avoid splashing the orifice or outer surfaces; close containers.
```